QUEST_LV_0100.tsv

```diff
@@ -626,16 +626,16 @@ QUEST_LV_0100_20150317_000625	$When the suppressed power flows yet again, the Gu
 QUEST_LV_0100_20150317_000626	$Abnormal Guardians are attacking the magical container.{nl}Protect the magical container until the suppressed power flows again.
 QUEST_LV_0100_20150317_000627	$The magical container is like the heart of the Royal Mausoleum.{nl}We can never let it be destroyed.
 QUEST_LV_0100_20150317_000628	$Order will surely return to this place.{nl}But it will take a long time.
-QUEST_LV_0100_20150317_000629	$Evil energies have condensed everywhere in the Mausoleum with the current of suppressed power.{nl}Find and remove the corrupted lumps.
+QUEST_LV_0100_20150317_000629	$An evil aura has condensed everywhere in the Royal Mausoleum with the current of suppressed power.{nl}Find and remove the corrupted lumps.
```

Comment:
Is it speaking of a 'current' ( like a river current, magic current of suppressed power )?
or the 'current status of things'
"due to the current suppressed power" / "due to the power being suppressed"